HeterogeneousCore/AlpakaTest/test/reader.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('Reader')
+
+process.source = cms.Source('PoolSource',
+    fileNames = cms.untracked.vstring('file:test.root')
+)
+
+process.load('FWCore.MessageService.MessageLogger_cfi')
```

Comment:
Just to note that it is not necessary anymore to explicitly load `MessageLogger_cfi` (the same configuration comes as part of `Process` constructor).

Reply:
Thanks for the information, I've simplified the configuration files accordingly and taken the opportunity to add some comments.